lib/src/dtos.dart

```diff
@@ -5,23 +5,27 @@ class CaptionTrackDto {
     required this.baseUrl,
     required this.name,
     required this.languageCode,
+    required this.kind,
   });
 
   factory CaptionTrackDto.fromJson(Map<String, dynamic> json) {
     return CaptionTrackDto(
       baseUrl: json['baseUrl'],
       name: json['name'] == null ? '' : json['name']['simpleText'],
       languageCode: json['languageCode'],
+      kind: json['kind'] == null ? '' : json['kind'],
```

Comment:
Would prefer to have `kind` as nullable instead of this (in case it is, indeed nullable)